build.sh

```diff
@@ -141,8 +141,12 @@ build_managed_corefx()
 
 build_native_corefx()
 {
+    # Format the Native Code to make sure it is format-compliant
+    echo "Formatting native source code..."
+    "$__nativeroot/format-code.sh"
```

Comment:
You're right, just did a test and modification times are not altered and incremental build show up-to-date. :)